### ISSUE TITLE
Backbone changes 

### ISSUE THREAD

**Issue by abhigoku10** [chufengt/iccv19_attribute#57]
@chufengt  i had few queries on the backbone
1. what was the reason of choice for using BN inception, since when i enable BNinception their is a boost in  accuracy but since its heavier we get a drop in speed 
2. which backbone would be a better choice that can balance the trade-off between accuracy and speed ??
3. which backbone is suitable for ease of integration of ALM with the backbone ?
4. what are the bottlenecks in the current backbone used ?

**Comment by chufengt** [comment]
1. just for convenience, some previous works used this backbone
2. i didn't try other backbones, you can see some new models which can achieve better trade-offs, e.g. efficientnet?
3. any backbone can use ALM
4. didn't catch your point, but you can see bn-inception original papers

**Comment by abhigoku10** [comment]
@chufengt  1,2,4 thanks for the response 
3. when backbone is changed should be modify the losses aspects accordingly or existing ALM loss equation works for this

**Comment by chufengt** [comment]
just keep the loss

**Comment by abhigoku10** [comment]
@chufengt  when using pretrained=True are you freezing only the last layer and training or retraining from begining

**Comment by chufengt** [comment]
no freezing, just for parameter initialization

**Comment by abhigoku10** [comment]
@chufengt  when we change the backbone to resent without including CAM and STN , does it work since we are providing annotation in a label format ie 1,0. so the main advantage of using label vector annotation with STN is for this reason right ??

**Comment by chufengt** [comment]
using ResNet only (w/o STN) can produce a reasonable result (see https://github.com/aajinjin/Strong_Baseline_of_Pedestrian_Attribute_Recognition)
label vector annotation? it's just for convenience, nothing to do with STN or backbones
